app/src/renderer/css/main.css

```diff
@@ -26,7 +35,7 @@
   border-right: 1rem solid transparent;
   border-bottom: 1rem solid $background-light;
   border-left: 1rem solid transparent;
-  transition: all .3s ease;
+  transition: border-bottom-width .2s ease, border-bottom-color .3s ease;
```

Comment:
Left the `300ms` on `border-bottom-color` intentionally, to not break any existing animation that I'm not aware of.